gc3libs/exceptions.py

```diff
@@ -483,6 +482,13 @@ class ApplicationDescriptionError(FatalError):
     """
     pass
 
+class SpoolDirError(InvalidValue, UnrecoverableError):
```

Comment:
This should probably be raised by line s451--452 in `batch.py` and the equivalent in `shellcmd.py`